pages/support.md

```diff
@@ -0,0 +1,75 @@
+<script>
+{
+	"title": "jQuery Support"
+}
+</script>
+
+## Community support
+
+The following learning resources are available online and free of charge:
+
+* [jQuery Learning Center](https://learn.jquery.com/)
+* [jQuery API Documentation](https://api.jquery.com/)
+* [jQuery UI Demos](https://jqueryui.com/demos/)
+* [jQuery UI API Documentation](https://api.jqueryui.com/)
+
+### Matrix chat
+
+jQuery is on Matrix! The support channel for jQuery is [#jquery_meeting:gitter.im](https://app.element.io/#/room/#jquery_meeting:gitter.im). You can [read the channel on Element](https://app.element.io/#/room/#jquery_meeting:gitter.im) without an account, or [join the channel](https://matrix.to/#/#jquery_meeting:gitter.im) via any Matrix client.
+
+We use the public [#jquery_dev:gitter.im channel](https://app.element.io/#/room/#jquery_dev:gitter.im) to discuss project developments, and also hold [weekly meetings](https://meetings.jquery.org/).
+
+### IRC chat
+
+The IRC support channel for jQuery projects is `#jquery` on [Libera Chat IRC](https://libera.chat/). You can [join via webchat](https://web.libera.chat/#jquery), or via an IRC client.
```

Comment:
I'm not sure I'd mention the IRC channel anymore.

Reply:
That can't be true. Currently, there's only 40 people in IRC and 916 in Matrix.

Reply:
Whatever the case, I think we want to direct people to Matrix and way from IRC.

Reply:
The Matrix room is migrated from Gitter. While Matrix is slightly more chatroom-like than Gitter (in that people do use join and leave buttons on Matrix), for Gitter it was afaik almost unheard off for anyone to leave a channel, and also rarely desired since it wasn't something people got notified through from apps or anything. Anyone who ever clicked on a Gitter button and auth'ed with GitHub is permanently in the room they clcked on, even if they never actually open the Gitter website after that.

IRC on the other hand is strictly session-based, i.e. who has the webchat or other client open at this moment (except for those who use bouncers or IRCCloud).

I wasn't counting stats or anything. I have the jquery Matrix room open and the before-last message is from "Sat, Aug 26, 2023". I also have #jquery joined in my IRCCloud and scrolling back there, nearly every day of the two months there's non-zero real-seeming poeple with real-seeming names sending real-seeming messages. I don't see any spam in recent history, and I have meta events hidden in my client (i.e. joins, parts, and name changes).

In any event, I'm just the messenger (pun intended). Happy to leave it out, move it down, minimise it, or otherwise not promote it.

Reply:
Ok, then I'm wrong. With that many messages, we should keep it. Sounds like I should be in IRC.